apps/api/src/domain/models/vocabulary-list/entities/vocabulary-list.entity.ts

```diff
@@ -23,10 +21,19 @@ import { ADD_TERM_TO_VOCABULARY_LIST } from '../commands/add-term-to-vocabulary-
 import { TRANSLATE_VOCABULARY_LIST_NAME } from '../commands/translate-vocabulary-list-name/constants';
 import {
     CannotAddMultipleEntriesForSingleTermError,
+    CannotHaveTwoFilterPropertiesWithTheSameNameError,
     DuplicateVocabularyListNameError,
 } from '../errors';
+import { VocabularyListFilterPropertyMustHaveAtLeastOneAllowedValueError } from '../errors/vocabulary-list-filter-property-must-have-at-least-one-allowed-value.error';
+import { DropboxOrCheckbox } from '../types/dropbox-or-checkbox';
 import { VocabularyListEntry } from '../vocabulary-list-entry.entity';
-import { VocabularyListVariable } from './vocabulary-list-variable.entity';
+import { VocabularyListFilterProperty } from './vocabulary-list-variable.entity';
+
+// TODO break out this type
```

Comment:
Let's do that as part of [this story](https://www.pivotaltracker.com/story/show/186469235).